plugin/core/clients.py

```diff
@@ -53,7 +52,7 @@ def window_configs(window: sublime.Window) -> 'Dict[str, ConfigState]':
     if window.id() in clients_by_window:
         return clients_by_window[window.id()]
     else:
-        # debug("no configs found for window", window.id())
+        log(2, "no configs found for window %s", window.id())
         return {}
```

Comment:
Not sure why I uncommented this one. I can comment it back if it is too annoying log message.